src/precompile.jl

```diff
@@ -0,0 +1,4 @@
+function _precompile_()
+    ccall(:jl_generating_output, Cint, ()) == 1 || return nothing
+    precompile(GridLayout, (Int, Int))
```

Comment:
Is `precompile(GridLayout, (Int, Int))` the only thing you thought was worth doing, or would it make sense to precompile all functionality that the tests go over?

Reply:
Nah, just a conversation-starter :smile:.